client/src/main/java/com/vesoft/nebula/client/graph/ResultSet.java

```diff
@@ -105,9 +105,11 @@ public ResultSet(List<byte[]> columns, List<RowValue> rows) {
             this.columns.add(new String(column).intern());
         }
         this.rows = rows;
```

Comment:
need to make sure the `this.rows` is not null, if not, other places use it, it will use the null rows.

Reply:
use this it works:  this.rows = new ArrayList<>();